frontend/cypress/e2e/exercise.cy.ts

```diff
@@ -0,0 +1,897 @@
+            .firstElement()
+            .its('reachableHospitals')
+            .should('not.be.empty');
+
+        cy.get('[data-cy=transferPointPopupCloseButton]').click();
+        cy.dragToMap('[data-cy=draggableVehicleDiv]');
+        cy.dragToMap('[data-cy=draggablePatientDiv]');
+
+        cy.get('[data-cy=chooseTransferTargetPopupHospitalDropdown]')
+            .first()
+            .click();
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[Hospital] Transport patient to hospital'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('hospitals')
+            .itsValues()
+            .firstElement()
+            .its('patientIds')
+            .should('not.be.empty');
+
+        cy.wait(150); // Do not trigger a double click
+        cy.get('[data-cy=openLayersContainer]').click();
+        cy.get('[data-cy=transferPointPopupHospitalNav]').click();
+        cy.get('[data-cy=transferPointPopupRemoveHospitalButton]').click();
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[TransferPoint] Disconnect hospital'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('transferPoints')
+            .itsValues()
+            .firstElement()
+            .its('reachableHospitals')
+            .should('be.empty');
+
+        cy.get('[data-cy=trainerToolbarCreationButton]').click();
+        cy.get('[data-cy=trainerToolbarHospitalsButton]').click();
+
+        cy.log('delete a hospital');
+        cy.get('[data-cy="hospitalDeleteButton"]').click();
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should('have.property', 'type', '[Hospital] Remove hospital');
+
+        cy.getState().its('exerciseState').its('hospitals').should('be.empty');
+    });
+
+    it('can manage transfer points and transfer vehicles', () => {
+        cy.dragToMap('[data-cy=draggableTransferPointDiv]');
+
+        cy.get('[data-cy=openLayersContainer]').click();
+        cy.get('[data-cy=transferPointPopupInternalNameInput]')
+            .clear()
+            .type('ABC123');
+
+        cy.get('@trainerSocketPerformedActions')
+            .lastElement()
+            .should(
+                'have.property',
+                'type',
+                '[TransferPoint] Rename TransferPoint'
+            );
+
+        cy.getState()
+            .its('exerciseState')
+            .its('transferPoints')
+            .itsValues()
+            .firstElement()
+            .should('have.property', 'internalName', 'ABC123');
+
+        cy.get('[data-cy=transferPointPopupCloseButton]').click();
+        cy.dragToMap('[data-cy=draggableTransferPointDiv]');
+
+        cy.wait(150);
```

Comment:
Same here

Reply:
Fixed in b501980e368668c216da57909d2ea74c919bebe8

Reply:
Edit: Fixed in 483896ce042712d2fa69f3896bca3e1e5fd2d7ec